flocker/node/_change.py

```diff
@@ -77,7 +84,10 @@ def run_state_change(change, deployer):
             )
         return d
 
-    return change.run(deployer)
+    with change.eliot_action.context():
```

Comment:
I wonder if it's worth having a utility function `eliot.twisted.run()` that does that? If so, file an issue.